python/ray/tests/test_actor_advanced.py

```diff
@@ -1175,6 +1175,61 @@ def condition2():
     ray.shutdown()
 
 
+def test_actor_timestamps(ray_start_regular):
```

Comment:
How does this work for actors with `max_restarts > 0`? Seems like the behavior is going to be pretty strange... please add a test case for it as well

Reply:
Intended behavior is to reflect the uptime of the actor handle (if the actor transparently restarts, we shouldn't reset the start time). 

I'll add a test for it.